src/data/quiz-data.js

```diff
@@ -0,0 +1,44 @@
+import { v4 as uuid } from 'uuid';
+
+export const quiz = [
+  {
+    _id: uuid(),
+    question:
+      "Lorem ipsum dolor, sit amet consectetur adipisicing elit. ullam eveniet, laudantium harum! Quam, eos esse?",
+    option1: { option:"option 1", answer:true },
+    option2: { option:"option 2", answer:false },
+    option3: { option:"option 3", answer:false }
+  },
```

Comment:
suggestion - later you can use APIs for fetching quiz data instead of storing data in array

Reply:
yes i will make the changes later, when i working on functionality